freeipa/src/main/resources/freeipa-salt/salt/freeipa/backup-full.sls

```diff
@@ -0,0 +1,13 @@
+include:
+  - freeipa.backup-common
+
+{% if salt['pillar.get']('freeipa:backup:enabled') %}
+freeipa_full_backup:
+  cmd.run:
+    - name: /usr/local/bin/freeipa_backup -t FULL -f "{{salt['grains.get']('fqdn')}}/full" && echo $(date +%Y-%m-%d:%H:%M:%S) >> /var/log/freeipa_full_backup-executed
```

Comment:
A more general question: I see that freeipa_backup script does a lock. So if hourly backup is running when full backup is requested for the same node, then full backup would die.
- What happens if a DATA backup is running on one node and a full backup on another: does DATA backup also stop freeipa services? How much more lightweight is a DATA backup than a FULL one?
- If full backup coincides with regular backup on a node, then full fails: that also fails the upgrade. Would it make sense to wait for the regular backup to end and then carry on the upgrade?

Reply:
- DATA or incremental backup doesn't stop any services
- as it's incremental it's faster and smaller in size
- only one backup can run on an instance, the other one would fail because of the lock
- retry logic should take care of that in FMS. An incremental backup in FreeIPA is a few sec operation, full backup is less than a minute. No upgrade should fail because of a backup collision. And if it fails it still no big issue, the customer could initiate it again.